README.md

```diff
@@ -127,6 +127,10 @@ Recently we have been actively improving our query engine primarily for better c
 
 Please refer to the [SQL Language Reference Manual](./docs/user/index.rst), [Piped Processing Language (PPL) Reference Manual](./docs/user/ppl/index.rst) and [Technical Documentation](https://opensearch.org/docs/latest/search-plugins/sql/index/) for detailed information on installing and configuring plugin.
 
+## Forum
+
+For additional help with the plugin, including questions about opening an issue, try the OpenSearch [Forum](https://forum.opensearch.org/c/plugins/sql/8).
```

Comment:
Do, or do not.  There is no try.  

How about: "**visit** the OpenSearch [Forum]"?

Reply:
Have a look how it is done on other OpenSearch repos.
Not a best example, but most familiar for me: https://github.com/opensearch-project/opensearch-net#project-resources

Reply:
@acarbonetto @Yury-Fridlyand There isn't perfect uniformity across the various plugin README.md files. Approximately half include the "chat – forum" icon and link at the very top. This may or may not change. So I'm leaving this particular file as is for the time being. But it's formally correct to have the new "Forum" heading in the TOC. Thanks for the suggestion. The tone of the "try Forum" language is gradually disappearing from documentation. So in spite of consistency, I changed to "visit". Thanks to all for your time. Please merge when there's a free moment.